tests/executables_src/testLMapTagModifier.cc

```diff
@@ -42,11 +45,14 @@ BOOST_AUTO_TEST_CASE(testSet) {
 
   device.open("(logicalNameMap?map=tagModifierPlugin.xlmap)");
   auto cat = device.getRegisterCatalogue();
+  auto baseInfo = cat.getRegister("plain");
+  BOOST_TEST(baseInfo.getTags().empty());
```

Comment:
This does not test that "set" replaces the existing tags. I would add a "PLAIN" tag to the plain register. Well, it's not plain any more, but it serves the purpose.